src/uu/stty/src/stty.rs

```diff
@@ -438,6 +449,18 @@ fn cc_to_index(option: &str) -> Option<SpecialCharacterIndices> {
     None
 }
 
+fn string_to_combo(arg: &str) -> Option<&str> {
+    let negate = arg.starts_with('-');
+    let name = arg.trim_start_matches('-');
+    for combo in COMBINATION_SETTINGS {
+        // combo must exist AND it has to be negatable if the user is attempting to negate it
+        if name == combo.0 && (!negate || combo.1) {
+            return Some(arg);
+        }
+    }
+    None
```

Comment:
While this works fine, I think you could make the intention clearer by using `find` and getting rid of `combo.0` and `combo.1`. Something like:
```rust
    COMBINATION_SETTINGS
        .iter()
        .find(|&&(combo_name, is_negatable)| name == combo_name && (!is_negated || is_negatable))
        .map(|_| arg)
```